zingolib/src/lightclient.rs

```diff
@@ -450,8 +450,10 @@ impl LightClient {
         }
     }
 
+    // pub async fn do_list_txsummaries(&self) -> (Vec<ValueTransfer>, Result<(), ZingoLibError>) {
```

Comment:
It seems like we're experimenting with the return type of this method?

Reply:
i will remove this structure from the PR